### DISCUSSION TITLE
SPM MAT Event File

### DISCUSSION THREAD

**Discussion by linjjiang** [HALFpipe/HALFpipe#437] [General]
Hi,

I'm having trouble entering the SPM .mat event file. I created the task event file following SPM's standard format, with three variables: names, onsets, and durations:
![image](https://user-images.githubusercontent.com/27575037/192652155-0e4ab39a-169a-4df4-aa8b-caf686ba5cde.png)

When I entered the file path into HALFpipe, it recognize the files correctly (e.g., "Found 6 files for 6 tasks"). However, after I specified units, the following error message occurred that prevented me from moving forward:
![image](https://user-images.githubusercontent.com/27575037/192651986-c62df32f-84c1-4456-be5a-55171e2dcd90.png)

Do you have any idea what the problem might be? Thank you!

Best,
Linjing

**Comment by HippocampusGirl** [comment]
Hi @Linjing95,

Could you provide me with an example .mat file so that I can further explore the problem? You can either upload it here, or e-mail it to me at lea@fmri.science :-)
